### DISCUSSION TITLE
Datasets for RAG/finetuning/etc.

### DISCUSSION THREAD

**Discussion by vladiliescu** [microsoft/community-content#5] [Season of AI]
Hi there, the demo videos reference several datasets for RAG, finetuning, etc. 

Are these published somewhere (couldn't find a link in the repo but maybe I haven't looked hard enough) or should we create these ourselves?

**Comment by toddanglin** [comment]
Thanks for the question, @vladiliescu. I presume you're talking about the demo videos for the Azure AI Studio talk?

@amynic @hnky Can you assist here?

**Reply by vladiliescu** [reply]
Ah yes, sorry, missed specifying that -- indeed I'm referring to the Getting Started with AI Studio videos

**Reply by nielsberglund** [reply]
@vladiliescu Did you ever get an answer to this?

Tagging @toddanglin , @amynic, @hnky

**Reply by vladiliescu** [reply]
@nielsberglund not yet, but it’s not a blocker — I’ve decided to skip those bits in my talk.

**Reply by nielsberglund** [reply]
Thanks, @vladiliescu! I found similar datasets for the first demo. However, I don't really know for the fine-tuning demo —I may skip that.

**Reply by swaminathanvetri** [reply]
@nielsberglund - I am in need of the same dataset to run through the demos for Azure AI studio. Would you mind sharing the links for the dataset you may have?

**Reply by amynic** [reply]
Hi everyone - my best recommendation would be to use this public repo sample called Contoso Chat - you will have everything you need to deploy and run (inc datasets) https://github.com/Azure-Samples/contoso-chat/tree/main

**Reply by GunaTeja777** [reply]
Thanks @amync — this is really helpful for RAG! Following up on the original ask though: are there similar sample datasets we could point to for the fine-tuning side of the demos, or is Contoso Chat the closest fit for that too?

**Comment by nielsberglund** [comment]
Hi all!

So, I used what Amy refers to.

Niels
